frontend/src/lib/components/details/recommendations.svelte

```diff
@@ -14,6 +14,8 @@
 
   let loadedPage: boolean
 
+  register()
```

Comment:
huh, wat dis?

Reply:
They have removed Svelte components in v9. So we have to use the swiper element web component. Then we have to register the swiper custom elements.

See: https://swiperjs.com/element